content/dogepedia/articles/becoming-a-dogecoin-developer.md

```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-How can shibes get involved with Dogecoin development?
+Dogecoin is an open-source project, meaning that anyone with some technical know-how can contribute to Dogecoin’s technical development. Even if you’re new to the space: it’s never too late to learn new tricks!  

```

Comment:
`Even if you're new to the space, it's never too late to learn new tricks!` or perhaps a semi-colon instead of colon?